### ISSUE TITLE
Document accessibility testing workflow

### ISSUE THREAD

**Issue by vsalanga** [WizyEMM/wizyemm-automation-testing#165]
Write up how to add new a11y checks, interpret violation reports, and triage/exempt known issues.
